### PR TITLE
Adds skeleton loader to workflows

### LINKED ISSUE #4124: Skeleton Loader for Workflows
Add skeleton loader to workflows

### LINKED ISSUE #4364: skeletons not same size
skeleton needs to be bigger:

image

loaded:

image

### PR DESCRIPTION
**Pull request by CarinaWolli**
## What does this PR do?

- Adds Skeleton loader to workflows ([Loom Video](https://www.loom.com/share/efcfc1bebd3f4248a2cb1c7c722881bf))
- Adds missing 6px border-radius to other skeleton loaders
- Some UI fixes in workflows
- Fixes design of booking skeleton loader 

Fixes #4124
Fixes #4364

### CODE REVIEW

File: apps/web/components/eventtype/SkeletonLoader.tsx [hunk 1]
```diff
@@ -5,13 +5,13 @@ function SkeletonLoader() {
   return (
     <SkeletonContainer>
       <div className="mb-4 flex items-center">
-        <SkeletonAvatar className="h-8 w-8" />
+        <SkeletonAvatar width="8" height="8" />
```
Comment (sean-brydon) [inline apps/web/components/eventtype/SkeletonLoader.tsx:8]: If im not mistaken this is bad practise and we should be using the w/h method in classNames - @zomars recently made a change to support this

**Review by sean-brydon — DISMISSED** [review]

File: apps/web/components/eventtype/SkeletonLoader.tsx (hunk 1 above)
Comment (zomars) [reply apps/web/components/eventtype/SkeletonLoader.tsx:8]: It's ok since this is a v1 view

Comment (zomars) [reply apps/web/components/eventtype/SkeletonLoader.tsx:8]: Will be deprecated anyways

**Review by zomars — APPROVED** [review]
LGTM

**Pull request merged**